src/app/app-routing.module.ts

```diff
@@ -87,6 +92,31 @@ const routes: Routes = [
             path: 'categories/new',
             component: CategoryDetailComponent
           },
+          // --------------SUB-CATEGORIES--------------
+          {
+            path: 'subcategories',
+            component: SubcategoriesComponent,
```

Comment:
You should have a default route. When the user clicks the sub-categories option, they should see the operators by default. Check how it's done for the fields.